File: python/calculate_mode.py
def calculate_mode(lista):
    freq = {}
    for i in lista:
        freq[i] = freq.get(i,0) +1
    # sortedfreq = {x:y for x,y in sorted(freq.items(), key=lambda item:item[1],reverse=True)}
    
    maxfreq = max(freq,key=freq.get)
    result = [i for i in freq if freq.get(i) == freq.get(maxfreq)]
    
    return result
    

File: js/calculateMode.js
exports.calculateMode = (lista) => {
    let freq = new Map()
    for (i of lista) {
        freq.set(i,0)
    }
    
    for (x of lista){
        let change = freq.get(x) + 1
        freq.set(x,change)
    }
    
    const maxi = Math.max(...freq.values())
    
    result = []

    for (let [key,value] of freq) {
        if (value === maxi)  {
            result.push(key)
        }
    }
    c
    return result
}
